sessionctx/variable/sysvar.go

```diff
@@ -2751,6 +2751,12 @@ var defaultSysVars = []*SysVar{
 			return nil
 		},
 	},
+	{Scope: ScopeGlobal, Name: TiDBEnableCheckConstraint, Value: BoolToOnOff(DefTiDBEnableCheckConstraint), Type: TypeBool, SetGlobal: func(ctx context.Context, vars *SessionVars, s string) error {
```

Comment:
@CbcWestwolf PTAL at if this variable is defined and used correctly

Reply:
Yes, it is the way we use it.